Source/WebDriver/WebSocketServer.cpp

```diff
@@ -71,7 +70,7 @@ void WebSocketServer::removeConnection(const WebSocketMessageHandler::Connection
         m_connectionToSession.remove(it);
 }
 
-RefPtr<Session> WebSocketServer::session(const WebSocketMessageHandler::Connection& connection)
+RefPtr<Session> WebSocketServer::session(const WebSocketMessageHandler::Connection& connection, RefPtr<Session> existingSession)
```

Comment:
This method is super confusing. it receives a session and it always returns that session or nullptr. I guess what we want here is checking the service session is handled by the server? If that's the case, I think this could be renamed as checkSession() or something like that and return a bool. Another option would be to return the session id, renaming it was sessionID. then the caller can check if it matches whatever session.

Reply:
Thanks for the suggestion. I'm going with the approach of returning the `SessionID`.